app/javascript/packages/phone-input/index.ts

```diff
@@ -23,10 +25,8 @@ interface IntlTelInput extends IntlTelInputPlugin {
 }
 
 const isPhoneValid = (phone, countryCode) => {
-  let phoneValid = isValidNumber(phone, countryCode);
-  if (!phoneValid && countryCode === 'US') {
-    phoneValid = isValidNumber(`+1 ${phone}`, countryCode);
-  }
+  const phoneValid = isValidNumber(phone, countryCode);
```

Comment:
If this method is now just a proxy to `isValidNumber`, should we get rid of it and call `isValidNumber` directly?

Reply:
yup, just took care of it in the following commit